2-multitenant/modules/env_baseline/main.tf

```diff
@@ -133,6 +132,23 @@ module "gke" {
   monitoring_enabled_components        = ["SYSTEM_COMPONENTS", "DEPLOYMENT"]
 
   remove_default_node_pool = true
+  cluster_autoscaling = {
+    enabled             = var.cluster_type == "STANDARD-NAP" ? true : false
+    autoscaling_profile = "BALANCED"
+    max_cpu_cores       = 100
+    min_cpu_cores       = 0
+    max_memory_gb       = 1024
+    min_memory_gb       = 0
+    gpu_resources = [
+      {
+        resource_type = "nvidia-tesla-t4"
+        minimum       = 0
+        maximum       = 4
+      }
+    ]
```

Comment:
gpu_resources is added just for STANDARD_NAP? is it intended?

Reply:
At least currently, as we don't currently have a workload that requires the gpus and so would scale to 0.